pymeasure/instruments/siglenttechnologies/siglent_sds1072cml.py

```diff
@@ -0,0 +1,309 @@
+#
+# This file is part of the PyMeasure package.
+#
+# Copyright (c) 2013-2024 PyMeasure Developers
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+from pymeasure.instruments import Instrument, Channel
+from pymeasure.instruments.validators import strict_discrete_range,truncated_discrete_set,strict_range,truncated_range
+import struct
+
+class VoltageChannel(Channel):
+
+    """ Implementation of a SIGLENT SDS1072CML Oscilloscope channel
+
+    """
+    vertical_division=Channel.control(
+        "C{ch}:VDIV?","C{ch}:VDIV %s",
+        "Sets or gets the vertical sensitivity of a channel.",
```

Comment:
Docstrings in general should be in imperative mode (so "Set or Get"...)

We in pymeasure use "Control" for "Set or get" as the first word of a `control`. For a `measurement` it is "Measure" or "Get" and for a `setting` it is "Set".

```suggestion
        "Control the vertical sensitivity of a channel.",
```